src/main/daml/Daml/Finance/Claims/Util/Builders.daml

```diff
@@ -169,12 +169,16 @@ createFxAdjustedPrincipalClaim dateToTime ownerReceives fxRateMultiplier notiona
 
 -- | Create the claim for a long vanilla option (cash-settled, automatically exercised).
 createVanillaOptionClaim : (Date -> Time) -> Decimal -> Observable -> Deliverable -> Date -> Bool
-  -> C
-createVanillaOptionClaim dateToTime strike spot cashInstrument expiryDate isCall =
+  -> Bool -> Decimal -> C
+createVanillaOptionClaim dateToTime strike spot cashInstrument expiryDate isCall
+  percentageOfStrike notionalMultiplier =
```

Comment:
Could you help me understand what `percentageOfStrike` means?

Reply:
It indicates whether the option claim should be on an absolute scale (e.g. 3 EUR) or a percentage of strike (e.g. 3 EUR / 60 EUR = 5%).

Reply:
Thanks! Is it common to define the payoff as percentage of strike?

I am used to seeing the strike defined in terms of the spot price at trade inception, but first time I see this :)

Reply:
I thought about that as well. I did not want to call it percentageOfSpot, since spot changes over time. I think spot price at trade inception coincides with the strike for some instruments, like the BRC. Happy to rename it though, if you have a suggestion.

Reply:
As discussed, I changed this to `referenceLevel` instead.